doc/api/process.markdown

```diff
@@ -873,6 +883,38 @@ a diff reading, useful for benchmarks and measuring intervals:
     }, 1000);
 
 
+## process.send(message[, sendHandle])
+
+* `message` {Object}
+* `sendHandle` {Handle object}
+
+When node is spawned with an IPC channel attached, it can send messages to its
+parent process using `process.send()`. Each will be received as a
+['message'](child_process.html#child_process_event_message)
```

Comment:
Documentation references should be in this format I think `[text][]` and then at the bottom of the page, `[text]: relative url`. PTAL at #2142 for reference.